### PR TITLE
align navbar items

### PR DESCRIPTION
**Pull request by Aimanosi**
**What kind of change does this PR introduce?**

Eliminate huge white space on the Organization navbar by aligning all nav items to the center when the screen size is <=1199px and remove repetitive organization image at the bottom of the navbar and replace it with text "more" close to the drop down arrow.

**Issue Number:**

- #748 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

solution

**Does this PR introduce a breaking change?**

No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by Aimanosi** [comment]
HI @palisadoes please review this PR and let me know if you need me to do anything else.

File: src/components/AdminNavbar/AdminNavbar.tsx [hunk 1]
```diff
@@ -190,11 +190,7 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
                 data-testid="logoutDropdown"
               >
                 {data?.organizations[0].image ? (
-                  <img
-                    src={data?.organizations[0].image}
-                    className={styles.roundedcircle}
```
Comment (palisadoes) [inline src/components/AdminNavbar/AdminNavbar.tsx:195]: Why did you remove this image? Won't we need it for wider screens?

**Review by palisadoes — CHANGES_REQUESTED** [review]

File: src/components/AdminNavbar/AdminNavbar.tsx (hunk 1 above)
Comment (Aimanosi) [reply src/components/AdminNavbar/AdminNavbar.tsx:195]: I don't think we need the image on the right side of the navbar even for the wider screen because we already have it on the left side, the repetition is unnecessary and it doesn't really serve any purpose, replacing that image with with a span element that says "more" serves two purposes.

1. It tells the user that there is more to explore on that side of the navbar.
2. And it eliminates the repetition of same image.

navbar wide screen

**Review by palisadoes — APPROVED** [review]

File: src/components/AdminNavbar/AdminNavbar.tsx [hunk 2]
```diff
@@ -190,11 +190,7 @@ function AdminNavbar({ targets, url_1 }: NavbarProps): JSX.Element {
                 data-testid="logoutDropdown"
               >
                 {data?.organizations[0].image ? (
-                  <img
-                    src={data?.organizations[0].image}
```
Comment (palisadoes) [inline src/components/AdminNavbar/AdminNavbar.tsx:194]: 1. Using "More" now requires translations.
2. Can you think of a way in which we can use a image that is suitable for this operation that could be used?

**Review by palisadoes — CHANGES_REQUESTED** [review]

File: src/components/AdminNavbar/AdminNavbar.tsx (hunk 2 above)
Comment (Aimanosi) [reply src/components/AdminNavbar/AdminNavbar.tsx:194]: I did a quick research and the most suitable representation I could find are icons, I couldn't find an image that could fit into our design and theme, if we must use an image it may have to be custom made for this project, my only problem with replacing "More" with an icon is that we already have an icon there, an alternative is that we remove "More" and increase the size of our existing icon.
icons

Comment (palisadoes) [reply src/components/AdminNavbar/AdminNavbar.tsx:194]: What about a hamburger icon?
image

Comment (Aimanosi) [reply src/components/AdminNavbar/AdminNavbar.tsx:194]: We would have a problem of duplicate icons displaying on screens <=1199, will this be acceptable?

ham-burger

Comment (Aimanosi) [reply src/components/AdminNavbar/AdminNavbar.tsx:194]: I think it might look like an error.

Comment (palisadoes) [reply src/components/AdminNavbar/AdminNavbar.tsx:194]: But it would have been there before, so the user would know the functionality. As you seem to be online let's continue this on slack.

Comment (Aimanosi) [reply src/components/AdminNavbar/AdminNavbar.tsx:194]: Okay, ill send you a message now.

**Review by palisadoes — APPROVED** [review]

**Comment by Aimanosi** [comment]
Hi @palisadoes sorry I had to redo the PR, I had to rectify a failed test, please review.

File: src/components/AdminNavbar/AdminNavbar.test.tsx [hunk 3]
```diff
@@ -189,48 +189,4 @@ describe('Testing Admin Navbar', () => {
...
-
-  //   const imageOptions = screen.getByTestId(/navbarOrgImagePresent/i);
-  //   const imageLogo = screen.getByTestId(/orgLogoPresent/i);
-  //   expect(imageLogo).toBeInTheDocument();
-  //   expect(imageOptions).toBeInTheDocument();
-  // });
-
-  test('Should check if organisation image is not present', async () => {
-    const { container } = render(
-      <MockedProvider addTypename={false} link={link1}>
```
Comment (palisadoes) [inline src/components/AdminNavbar/AdminNavbar.test.tsx:218]: We should be checking for the presence of the new image.

**Review by palisadoes — CHANGES_REQUESTED** [review]
We should be checking for the presence of the new image.

File: src/components/AdminNavbar/AdminNavbar.test.tsx (hunk 3 above)
Comment (Aimanosi) [reply src/components/AdminNavbar/AdminNavbar.test.tsx:218]: We have changed the image to a static navbar hamburger icon so there was no need to check the presence because it will always be there and  the functionality of the icon is limited to only toggling a menu. As such, there may be little value in writing automated tests specifically for this icon.

Alternatively I could wrap a stand alone icon in a span like I did with "More" and bring back the test, I just didn't think it was necessary.
please let me know what you think.

**Review by palisadoes — APPROVED** [review]
